content/en/continuous_integration/setup_tests/javascript.md

```diff
@@ -250,9 +257,7 @@ If you are running tests in non-supported CI providers or with no `.git` folder,
 [Mocha >=9.0.0][8] uses an ESM-first approach to load test files. That means that if ES modules are used (for example, by defining test files with the `.mjs` extension), _the instrumentation is limited_. Tests are detected, but there isn't visibility into your test. For more information about ES modules, see the [NodeJS documentation][9].
 
 ### Browser tests
-The JavaScript tracer does not support browsers, so browser tests with `mocha` or `jest` do not provide visibility on the browser. For tests with `cypress`, `dd-trace` provides visibility on the node process running the test, but not on the browser. 
-
-If you want visibility on the browser, consider [Real User Monitoring][10].
+Browser tests run with `mocha`, `jest`, `cucumber` and `cypress` are instrumented by `dd-trace`, but no visibility into the browser itself is provided (no network calls, user actions, page loads, etc.). If you want visibility into the browser process, consider [Real User Monitoring][10].
```

Comment:
```suggestion
Browser tests with `mocha`, `jest`, `cucumber` and `cypress` are instrumented by `dd-trace`, but no visibility into the browser itself is provided (no network calls, user actions, page loads, etc.). If you want visibility into the browser process, consider [Real User Monitoring][10].
```
This is a bit confusing because we often refer to "test run" as a concept

Reply:
maybe change to "browser tests executed"? Just `Browser tests with ...` does not sound right